Providers/Scripts/2.4x-2.5x/Scripts/nxOMSAgentNPMConfig.py

```diff
@@ -193,6 +194,9 @@ def Set(ConfigType, ConfigID, Contents, Ensure, ContentChecksum):
 def Test(ConfigType, ConfigID, Contents, Ensure, ContentChecksum):
     ShowMof('TEST', ConfigType, ConfigID, Contents, Ensure, ContentChecksum)
     retval = 0
+    if not os.path.exists(AGENT_SCRIPT_PATH):
```

Comment:
Good point. I thought it was copied by the NPM DSC module. 
Please include it in the https://github.com/Microsoft/PowerShell-DSC-for-Linux/blob/master/installbuilder/datafiles/Base_DSC.data similar to OMSRsyslog.post.sh
BTW, the owner should be root as it has sudo permission.